s3-lambda-comprehend-cdk-python/app.py

```diff
@@ -0,0 +1,113 @@
+#!/usr/bin/env python3
+import os
+import aws_cdk as cdk
+from aws_cdk import (
+    Duration,
+    Stack,
+    aws_iam as iam,
+    aws_lambda as lambda_,
+    aws_s3 as s3,
+    aws_lambda_event_sources as eventsources,
+    aws_dynamodb as dynamodb,
+    CfnOutput,
+)
+from constructs import Construct
+from aws_cdk.aws_dynamodb import (
+    Attribute,
+    AttributeType,
+    Billing,
+    TableV2,
+    TableEncryptionV2,
+)
+
+DIRNAME = os.path.dirname(__file__)
+
+
+class S3LambdaComprehendServerless(Stack):
+    def __init__(self, scope: Construct, construct_id: str, **kwargs) -> None:
+        super().__init__(scope, construct_id, **kwargs)
+
+        self.user_input_bucket = s3.Bucket(
+            self,
+            "s3-upload-input-text",
```

Comment:
S3 bucket names are globally unique. This will lead to errors as the bucket already exists. Consider making this a parameter or instructing the user to change it before deployment.

Reply:
Updated app.py with a comment asking user to replace the bucket name with a globally unique name.